src/onegov/agency/templates/macros.pt

```diff
@@ -7,18 +7,70 @@
 <body>
 
     <metal:agency_card define-macro="agency_card" i18n:domain="onegov.agency">
-        <div class="agency-card agency-single-item" tal:define="memberships list(agency.memberships); children list(agency.children)">
+        <div class="agency-card agency-single-item" tal:define="memberships list(agency.memberships); children list(agency.children)" tal:attributes="class 'columns medium-12'">
```

Comment:
No need for `tal:attributes` since the content is not dynamic. Also, this conflicts with the already existing `class` attribute...

Reply:
got it